commander/src/utils/download.ts

```diff
@@ -88,5 +72,13 @@ export const extract = async (
 export const downloadAndValidate = async (url: string, cacheDir: string) => {
 	await download(url, cacheDir);
 	await download(`${url}.SHA256`, cacheDir);
-	await validateChecksum(`${url}.SHA256`, cacheDir);
+
+	const { fileName, fileDir } = getDownloadedFileInfo(url, cacheDir);
+	const dataBuffer = fs.readFileSync(`${fileDir}/${fileName}`);
```

Comment:
It looks like the whole file would be read into RAM. Since it is pretty large already wouldn't it be nicer to read it as a stream instead?